ci/ciimage/android/install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+
+source /ci/common.sh
+source /ci/env_vars.sh
+
+export DEBIAN_FRONTEND=noninteractive
+export LANG='C.UTF-8'
+
+apt-get -y update
+apt-get -y upgrade
+
+pkgs=(
+  git ninja-build python3-pip sdkmanager
+)
+
+apt-get -y install "${pkgs[@]}"
+
+install_minimal_python_packages
+
+# cleanup
+apt-get -y clean
+apt-get -y autoclean
+
+# sdk install
+
+set -x
+
+if [ -z $ANDROID_HOME -o -z $ANDROID_SDKVER ]; then
+    echo "ANDROID_HOME and ANDROID_SDKVER env var must be set!"
+    exit 1
+fi
+
+test -d ${HOME}/.android || mkdir ${HOME}/.android
+# there are currently zero user repos
+echo 'count=0' > ${HOME}/.android/repositories.cfg
+cat <<EOF >> ${HOME}/.android/sites-settings.cfg
+@version@=1
+@disabled@https\://dl.google.com/android/repository/extras/intel/addon.xml=disabled
+@disabled@https\://dl.google.com/android/repository/glass/addon.xml=disabled
+@disabled@https\://dl.google.com/android/repository/sys-img/android/sys-img.xml=disabled
+@disabled@https\://dl.google.com/android/repository/sys-img/android-tv/sys-img.xml=disabled
+@disabled@https\://dl.google.com/android/repository/sys-img/android-wear/sys-img.xml=disabled
+@disabled@https\://dl.google.com/android/repository/sys-img/google_apis/sys-img.xml=disabled
+EOF
+
+ANDROID_SDKMAJOR=`echo ${ANDROID_SDKVER} | awk -F '.' '{print $1}'`
```

Comment:
Forget that ``` ` ``` exists. It is obsolescent.

Exclusively use `$()`

```suggestion
ANDROID_SDKMAJOR=$(echo ${ANDROID_SDKVER} | awk -F '.' '{print $1}')
```